Todo .go

```diff
@@ -17,7 +17,7 @@ import (
 //and also save after the delete
 type Todo struct {
 	//and if you set the strtuct at ID to string and adapt the code for it
-	Id   int    `json:"id"`
```

Comment:
it is much easier to handle it as a string currently :) (but int would be the correct type)